src/lib/UI/Module/ContentTree/NodeFactory.php

```diff
@@ -301,7 +302,9 @@ private function buildNode(
      */
     private function supplyTranslatedContentName(Node $node, array $contentById): void
     {
-        $node->name = $this->translationHelper->getTranslatedContentName($contentById[$node->contentId]);
+        if ($node->contentId !== self::TOP_NODE_CONTENT_ID) {
+            $node->name = $this->translationHelper->getTranslatedContentName($contentById[$node->contentId]);
+        }
```

Comment:
```suggestion
        if ($node->contentId !== self::TOP_NODE_CONTENT_ID) {
            $node->name = $this->translationHelper->getTranslatedContentName($contentById[$node->contentId]);
        }
        
```
^ CS: Missing new line